### PR TITLE
docs(common): fixed readme ESP #370

### PR DESCRIPTION
**Pull request by fvives99**
Updated SS, removed versions and collaborators, and updated logo to be visible in dark mode

### What does this PR do?

- resolves https://github.com/eoscostarica/guide.eoscostarica.io/issues/370 and https://github.com/eoscostarica/guide.eoscostarica.io/issues/485

### Steps to test
1. Open readme, see new ss, deleted sections and updated logo

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Guías EOS Costa Rica
+# Guías para Desarrolladores de EOS Costa Rica
 ![](https://img.shields.io/github/license/eoscostarica/guias.eoscostarica.io) ![](https://img.shields.io/badge/code%20style-standard-brightgreen.svg) ![](https://img.shields.io/badge/%E2%9C%93-collaborative_etiquette-brightgreen.svg) ![](https://img.shields.io/twitter/follow/eoscostarica.svg?style=social&logo=twitter) ![](https://img.shields.io/github/forks/eoscostarica/guias.eoscostarica.io?style=social)
 
 Sitio de documentación para desarrolladores de EOS Costa Rica.
 
-![Guide front page](static/img/guide-page.png)
+![Guide front page](static/img/guide-pageNew.png)
```
Comment (xavier506) [inline README.md:6]: no te parece esta muy grande esta imagen?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 Sitio de documentación para desarrolladores de EOS Costa Rica.
 
-![Guide front page](static/img/guide-pageNew.png)
+![Guide front page](https://user-images.githubusercontent.com/55892352/179573831-2efe763b-cd54-43fa-81e3-aed00ceff191.png)
```
[the remark this answers is not in the document]
Comment (fvives99) [reply README.md]: noted, will fix asap

File: README.md [hunk 3]
```diff
@@ -128,8 +120,8 @@ Si encuentra un error, informe los errores grandes y pequeños [**abriendo un is
 
 <p align="center">
 	<a href="https://eoscostarica.io/">
-		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-horizontal-transparent-white.png"
-		width="400px" >
+		<img src="https://user-images.githubusercontent.com/5632966/178800859-e05a07ff-b3e5-41a9-9137-d1a271b958ff.png"
```
Comment (xavier506) [inline README.md:123]: lets source this image from the same location as the other logos `.github` folder

Comment (fvives99) [reply README.md]: fixed.

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**